scripts/dev/downloader.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+printDownloaderHelp(){
+cat << EOF
+    Usage: download.sh [URL] [OPTIONS]
+
+    Example: download.sh http://ci.openframeworks.cc/libs/file.zip -s
+
+    Options:
+    -s, --silent                Silent download progress
+    -h, --help                  Shows this message
+EOF
+}
+
+downloader() { 
+
+	if [ -z "$1" ]
+	then printDownloaderHelp; fi
+
+    SILENTARGS="";
+    if [ "$2" == "-s" ]; then
+    	`sh $CATALINA_HOME/bin/startup.sh`
```

Comment:
What is this doing? Won't it fail on other OSs but macos?

Reply:
Yeah totally confused about the startup and shutdown commands. seems related to Apache Tomcat?  
@danoli3 what is the use of those calls? Thanks!